skrifa/src/outline/mod.rs

```diff
@@ -608,6 +609,27 @@ impl<'a> OutlineGlyphCollection<'a> {
         }
     }
 
+    /// Returns true when the interpreter engine _must_ be used for hinting
+    /// this set of outlines to produce correct results.
+    ///
+    /// This corresponds so FreeType's `FT_FACE_FLAG_TRICKY` face flag. See
+    /// the documentation for that [flag](https://freetype.org/freetype2/docs/reference/ft2-face_creation.html#ft_face_flag_xxx)
+    /// for more detail.
+    ///
+    /// When this returns `true`, you should construct a [`HintingInstance`]
+    /// with [`HintingOptions::engine`] set to [`Engine::Interpreter`] and
+    /// [`HintingOptions::target`] set to [`Target::Mono`].
+    ///
+    /// # Performance
+    /// This digs through the name table and potentially computes checksums
+    /// so it may be slow. You should cache the result of this function if
+    /// possible.
+    pub fn require_interpreter(&self) -> bool {
```

Comment:
This name works well. Optional: If we go with `hinting-reliant` elsewhere, we could call it that here, too, i.e. `is_hinting_reliant()`

Reply:
Thanks for making these changes.